### PR TITLE
Add zookeeper example based on #42 from @robbiet480

### PR DESCRIPTION
**Pull request by bobrik**

### CODE REVIEW

File: example_configs/zookeeper.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+rules:
+  - pattern: "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\\d)><>(\\w+)"
+    name: "zookeeper_$2"
+    labels:
+      brokerId: "$1"
```
Comment (brian-brazil) [inline example_configs/zookeeper.yaml]: Per #42, this shouldn't be a label from instrumentation as it's really a target label.

By convention we also do `broker_id`.

Comment (bobrik) [inline example_configs/zookeeper.yaml]: Should I just drop it then?

Comment (brian-brazil) [inline example_configs/zookeeper.yaml]: Yes, there can only be one broker per server.

Comment (bobrik) [inline example_configs/zookeeper.yaml]: Done

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**